js/src/manifests/manifest.js

```diff
@@ -25,6 +52,18 @@
         _this.jsonLd = jsonLd;
       });
     },
+    initFromInfoJson: function(infoJsonUrl) {
+      var _this = this;
+      this.request = jQuery.ajax({
+        url: infoJsonUrl,
+        dataType: 'json',
+        async: true
+      });
+
+      this.request.done(function(jsonLd) {
+        _this.jsonLd = _this.generateInfoWrapper(jsonLd);
```

Comment:
The idea here is to swap out the manifest request for the info.json request and the actual manifest for a dummy manifest structure created as a wrapper around the info.json as its service. The dropped image is the one service on the one resource on the one image on the one canvas in the one sequence of the dummy manifest.